### PR TITLE
chore(CFN): Allow access to CI keystore resources from CI and release roles

### PR DESCRIPTION
**Pull request by lucasmcdonald3**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

# Check any applicable:
- [ ] Were any files moved? Moving files changes their URL, which breaks all hyperlinks to the files.

### CODE REVIEW

**Review by ajewellamz — APPROVED** [review]

**Review by josecorella — APPROVED** [review]

File: cfn/Public-ESDK-Python-CI.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "DDB Table and IAM Managed Policies/Role for AWS KMS Hierarchical Keyring Testing"
+
+Parameters:
+  TableName:
+    Type: String
+    Description: Test Table Name
+    Default: HierarchicalKeyringTestTable
```
Comment (texastony) [inline cfn/Public-ESDK-Python-CI.yml:8]: Blocking: Remove unused Parameter
```suggestion
```

File: cfn/Public-ESDK-Python-CI.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  KeyStoreTable:
+    Type: String
+    Description: Key Store Test Table Name
+    Default: KeyStoreTestTable
```
Comment (texastony) [inline cfn/Public-ESDK-Python-CI.yml:12]: Blocking: Remove unused Parameter
```suggestion
```

File: cfn/Public-ESDK-Python-CI.yml [hunk 3]
```diff
@@ -0,0 +1,67 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "DDB Table and IAM Managed Policies/Role for AWS KMS Hierarchical Keyring Testing"
```
Comment (texastony) [inline cfn/Public-ESDK-Python-CI.yml:2]:
```suggestion
Description: "IAM Managed Policies/Role for AWS KMS Hierarchical Keyring Testing"
```

**Review by texastony — CHANGES_REQUESTED** [review]

**Review by texastony — APPROVED** [review]

File: cfn/Public-ESDK-Python-CI.yml [hunk 4]
```diff
@@ -0,0 +1,59 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "IAM Managed Policies/Role for AWS KMS Hierarchical Keyring Testing"
+
+Parameters:
+  ProjectName:
+    Type: String
+    Description: A prefix that will be applied to any names
```
Comment (lucasmcdonald3) [inline cfn/Public-ESDK-Python-CI.yml:7]:
```suggestion
    Description: A prefix that will be applied to any resource names
```

Comment (lucasmcdonald3) [reply cfn/Public-ESDK-Python-CI.yml]: (just restarting CI)

**Pull request merged**